### PR TITLE
Update Tracy to eeffb6d2 (2022-08-02)

### PR DESCRIPTION
**Pull request by antiagainst**

### CODE REVIEW

File: build_tools/third_party/tracy/CMakeLists.txt [hunk 1]
```diff
@@ -139,11 +139,14 @@ target_link_libraries(IREETracyServer
...
-file(GLOB IMGUI_SOURCES ${TRACY_SOURCE_DIR}/imgui/*.cpp)
+file(GLOB IMGUI_SOURCES
+  ${TRACY_SOURCE_DIR}/imgui/*.cpp
+  ${TRACY_SOURCE_DIR}/profiler/src/imgui/*.cpp
+)
 add_library(IREETracyIMGUI
   ${IMGUI_SOURCES}
 )
-setup_cxx_options(IREETracyServer)
+setup_cxx_options(IREETracyIMGUI)
```
Comment (ScottTodd) [inline build_tools/third_party/tracy/CMakeLists.txt:149]: Can we also add a build with `-DIREE_BUILD_TRACY=ON` to presubmit? https://github.com/iree-org/iree/blob/main/build_tools/cmake/build_host_tools.sh seems like a reasonable place to put that (maybe if linux, and if deps are available...)

Comment (antiagainst) [reply build_tools/third_party/tracy/CMakeLists.txt:149]: Yeah. I'm not sure about the dev environment we have for various bots. Let me try with https://github.com/iree-org/iree/pull/9992.

Comment (ScottTodd) [reply build_tools/third_party/tracy/CMakeLists.txt:149]: Okay, that looks like it's unfortunately going to be a little tricky. Maybe let's just proceed and accept the spotty presubmit coverage?

**Review by ScottTodd — APPROVED** [review]

File: build_tools/third_party/tracy/CMakeLists.txt (hunk 1 above)
Comment (antiagainst) [reply build_tools/third_party/tracy/CMakeLists.txt:149]: +1. I've tested locally that the `iree-tracy-*` binaries can compile. So let's land this.

We are only gonna update Tracy likely quarterly; so as long as we be more careful each time it should be fine.

**Pull request merged**